css/index.css

```diff
@@ -71,66 +75,108 @@ p {
     margin: 0 auto;
 }
 
+/*Header Section Style*/
+.header{
+    margin-top: 20px;
+    align-items: flex-end;
+    display: flex;
+    justify-content: space-between;
+    margin-right: 5%;   
+}
+.header a{
+    text-decoration: none;
+    color: black;
+}
+.header-img{
+    margin-top: 40px;
+}
+
+/*Top Section Style*/
 .top-content {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
-    margin-bottom: 20px;
+    margin: 20px 0;
     border-bottom: 1px dashed black;
 }
-
 .top-content .text-container {
     width: 48%;
     padding: 0 1%;
     padding-bottom: 20px;  
 }
 
+/*Middle Section Style*/
 .middle-content {
     margin-bottom: 20px;
     border-bottom: 1px dashed black;
 }
-
 .middle-content h2 {
     padding: 0 2%;
     margin-bottom: 0;
 }
-
 .middle-content .boxes {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
 }
-
-.middle-content .boxes .box {
```

Comment:
The parent styles cascades down to the children.  You can leave off all the individual boxes.